src/redux/slices/connectionSlice.ts

```diff
@@ -46,6 +47,18 @@ const connection = createSlice({
         remoteStream: initialState.remoteStream,
       };
     },
+    setCallStateTrue: (state: ConnectionState) => {
+      return {
+        ...state,
+        hangup: true,
+      };
+    },
+    setCallStateFalse: (state: ConnectionState) => {
+      return {
+        ...state,
+        hangup: false,
+      };
+    },
```

Comment:
Esto es una sugerencia, pero no es necesario implementarlo:
```suggestion
    toggleCallState: (state: ConnectionState) => {
      return {
        ...state,
        hangup: !state.hangup,
      };
    },
```

Reply:
Agregado

Reply:
Nono, era más bien un comentario para futuro (otros proyectos jeje), pero funcionaba. Ahora tendrías que adaptarlo para que sea toggle en vez de true y false.

Reply:
Aa pero no lo usaste. Da lo mismo entonces.

Reply:
Jjaja si no entendi mucho, pero si quieres lo saco...no lo implementé

Reply:
Nop, la idea es reemplazar `setCallStateTrue` y el `setCallStateFalse` por el `toggleCallState`, pero fue solo un comentario porque eso implica hacer un cambio en los _interactors_ y en los componentes que llaman estos interactors.

Reply:
Asi que si, bórralo nomás

Reply:
Jaja ya si cache, lo cambio en un rato

Reply:
Subido